tensorflow/python/keras/backend_test.py

```diff
@@ -1736,5 +1736,34 @@ def test_batch_normalization(self):
     self.assertEqual(normed.shape.as_list(), [10, 3, 5, 5])
 
 
+class ControlOpsTests(test.TestCase):
+
+  def test_function_switch_basics(self):
+    pred = keras.backend.placeholder([], dtype='bool')
+
+    def false_fun():
+      return array_ops.constant(False)
+
+    with self.cached_session():
+      x2 = keras.backend.switch(pred, lambda: array_ops.constant(True), false_fun)
+      x1 = keras.backend.function(pred, x2)([True])
```

Comment:
The x1/x2 naming here is bit unclear to me, x2 here is a tensor and x1 is a value. Can we give them proper name?

Reply:
Sure, same is updated in the code.

Regards
Amit